integration/utils/tx_tracker.go

```diff
@@ -219,8 +227,12 @@ func (txt *TxTracker) statusWorker(ctx context.Context, workerID int, fclient *c
 			// don't put too much pressure on access node
 			time.Sleep(sleepAfterOp)
 
-			// put it back
-			txt.txs <- tx
+			select {
+			case txt.txs <- tx:
```

Comment:
This can still livelock in general, when # txn > maxCap.   is there a strong reason why we need to use a limited worker pool instead of just using a goroutine per txn?  if it's only for rate limiting, it may be better to switch to a goroutine per txn, and use a semaphore for rate limiting.